uaclient/entitlements/tests/test_base.py

```diff
@@ -17,6 +17,7 @@
 
 class ConcreteTestEntitlement(base.UAEntitlement):
 
+    help_doc_url = ""
```

Comment:
This should be `None` if we don't convert `help_doc_url` to have a default (in which case it should be removed).

Reply:
I fixed this on UAEntitlement base class to be
    help_doc_url = None  # type: str


This resolved the mypy workaround I was trying to add by declaring the base class as type string. No need to set this attr on each subclass.